app/main/containers/Search/index.js

```diff
@@ -109,7 +110,9 @@ class Search extends Component {
     // NOTE: when page refreshed (location.reload) componentWillUnmount is not called
     window.addEventListener('beforeunload', this.cleanup)
     this.electronWindow.on('show', this.focusMainInput)
+    this.electronWindow.on('show', this.updateElectronWindow)
     this.electronWindow.on('show', trackShowWindow)
+    this.electronWindow.on('move', this.moveElectronWindow)
```

Comment:
The `moved` event unfortunately works only in MacOS, that's why I used `move + debounce`.

Agree on the other two points :)